scripts/calculateRevenue/protocols/fonbnk/index.ts

```diff
@@ -56,6 +56,10 @@ export async function getUserTransactions({
             timestamp: blockTimestampDate,
           })
         }
+      } else {
+        console.log(
+          `Fonbnk transfer transaction missing one of the required fields: ${transaction}`,
```

Comment:
I don't think this will serialize correctly. IIRC this might just print `[object]` or something similar.